components/eam/src/physics/cam/ndrop.F90

```diff
@@ -110,9 +110,23 @@ subroutine ndrop_init
    character(len=8)    :: unit
    logical :: history_amwg         ! output the variables used by the AMWG diag package
    logical :: history_verbose      ! produce verbose history output
+   real(r8) :: arg        ! argument for erf test [dimensionless]
+
 
    !-------------------------------------------------------------------------------
 
+   arg = 1.0_r8
+   if (abs(0.8427_r8 - erf(arg))/0.8427_r8 > 0.001_r8) then
```

Comment:
what is the significance of `0.8427`? Can we create a `parameter` for this?

Reply:
Is this subroutine actually testing the error function at runtime?

Reply:
Here is a link to Balwinder's notes:
https://eagles-project.atlassian.net/wiki/spaces/Computation/pages/edit-v2/1623228417
If the link doesn't work -- I think this is related to the fact that some fortran compilers don't provide Fortran intrinsic erf functions, in which case cime might provide its own error function.  This is to ensure that there is an erf function defined that gives the answers the erf functions in the code are expecting.  It was previously in dropmixnuc but I just moved it to ndrop_init.

To answer the other question -- 0.8427 is just erf(1.0), code is just testing whether this is in fact what the erf function produces.  It could certainly be converted to a parameter.

Reply:
Thanks! C and C++ both define `erf` in their standard libraries, so we can probably skip this exercise in the port.